README.md

```diff
@@ -6,4 +6,253 @@
+  | 'BillPayment/Create'
+  | 'BillPayment/Update'
+  | 'Budget/Create'
+  | 'Budget/Update'
+  | 'Class/Delete'
+  | 'Class/Merge'
+  | 'Class/Create'
+  | 'Class/Update'
+  | 'CreditMemo/Delete'
+  | 'CreditMemo/Emailed'
+  | 'CreditMemo/Void'
+  | 'CreditMemo/Create'
+  | 'CreditMemo/Update'
+  | 'Currency/Create'
+  | 'Currency/Update'
+  | 'Customer/Delete'
+  | 'Customer/Merge'
+  | 'Customer/Create'
+  | 'Customer/Update'
+  | 'Department/Merge'
+  | 'Department/Create'
+  | 'Department/Update'
+  | 'Deposit/Delete'
+  | 'Deposit/Create'
+  | 'Deposit/Update'
+  | 'Employee/Delete'
+  | 'Employee/Merge'
+  | 'Employee/Create'
+  | 'Employee/Update'
+  | 'Estimate/Delete'
+  | 'Estimate/Emailed'
+  | 'Estimate/Create'
+  | 'Estimate/Update'
+  | 'Invoice/Delete'
+  | 'Invoice/Emailed'
+  | 'Invoice/Void'
+  | 'Invoice/Create'
+  | 'Invoice/Update'
+  | 'Item/Delete'
+  | 'Item/Merge'
+  | 'Item/Create'
+  | 'Item/Update'
+  | 'JournalCode/Create'
+  | 'JournalCode/Update'
+  | 'JournalEntry/Delete'
+  | 'JournalEntry/Create'
+  | 'JournalEntry/Update'
+  | 'Payment/Delete'
+  | 'Payment/Emailed'
+  | 'Payment/Void'
+  | 'Payment/Create'
+  | 'Payment/Update'
+  | 'PaymentMethod/Merge'
+  | 'PaymentMethod/Create'
+  | 'PaymentMethod/Update'
+  | 'Preferences/Update'
+  | 'Purchase/Delete'
+  | 'Purchase/Void'
+  | 'Purchase/Create'
+  | 'Purchase/Update'
+  | 'PurchaseOrder/Delete'
+  | 'PurchaseOrder/Emailed'
+  | 'PurchaseOrder/Create'
+  | 'PurchaseOrder/Update'
+  | 'RefundReceipt/Delete'
+  | 'RefundReceipt/Emailed'
+  | 'RefundReceipt/Void'
+  | 'RefundReceipt/Create'
+  | 'RefundReceipt/Update'
+  | 'SalesReceipt/Delete'
+  | 'SalesReceipt/Emailed'
+  | 'SalesReceipt/Void'
+  | 'SalesReceipt/Create'
+  | 'SalesReceipt/Update'
+  | 'TaxAgency/Create'
+  | 'TaxAgency/Update'
+  | 'Term/Create'
+  | 'Term/Update'
+  | 'TimeActivity/Delete'
+  | 'TimeActivity/Create'
+  | 'TimeActivity/Update'
+  | 'Transfer/Delete'
+  | 'Transfer/Void'
+  | 'Transfer/Create'
+  | 'Transfer/Update'
+  | 'Vendor/Delete'
+  | 'Vendor/Merge'
+  | 'Vendor/Create'
+  | 'Vendor/Update'
+  | 'VendorCredit/Delete'
+  | 'VendorCredit/Create'
+  | 'VendorCredit/Update'
+```
+
+Events require that an integration webhook is configured in Quickbooks. 
+
+The connector triggers events of the following type:
+
+```typescript
+interface QBEvent {
+  realmId: string
+  name: string // Entity type name e.g. Customer, Account, Bill
+  id: string   // Entity ID
+  operation: string 
+  lastUpdated: string
+  action: string
+}
+```
+
+_Example:_
+
+```typescript
+quickbooksConnector.on({ type: 'Bill/Update' }, async (event, app) => {
+  console.log('Bill/Update event ')
+  console.log(event.id)
+  console.log(event.name)
+  console.log(event.operation)
+})
+```
+
+The description of fields and events can be found [here](hhttps://developer.intuit.com/app/developer/qbo/docs/develop/webhooks/entities-and-operations-supported)
+
+### <a name="actions"></a> Actions
+
```

Comment:
Add a comment about what you can do as actions.
Something explaining it's all through the sdk and ideally a link to the sdk to see the list of methods (see https://github.com/reshufflehq/reshuffle-aws-connectors/blob/master/doc/AWSSQSConnector.md#sdk-action)